chromaui/chromatic-e2e
# Replace Codacy code coverage reporting with Codecov

Issue: #

## What Changed

This removes our code coverage reporting to Codacy and instead sends it over to Codecov.
## How to test

Go to the Codecov page for this repo and see the test coverage reported.

Closes AP-5429
